### PR TITLE
Milestone-3/Blog app - create data model

### PR DESCRIPTION
**Pull request by edwardoluyara**
Here in this milestone I have been able to implement the following:
- Build project with schema
  - Create and run the necessary migration files. 
  - Include foreign keys
  - All columns that are foreign keys have corresponding indexes.

### CODE REVIEW

File: db/migrate/20220426235329_create_posts.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+
+      t.timestamps
+    end
+  end
+end
```
Comment (Whoistolu) [inline db/migrate/20220426235329_create_posts.rb]: - [ ] [REQUIRED] Kindly note that the `Post` table according to the ERD diagram has the `comments_counter` and the `likes_counter`. Kindly implement this. 
- [ ] [REQUIRED] Kindly add the necessary foreign key which will create `author_id`, and a corresponding index. The foreign key is on the ERD diagram.

File: db/migrate/20220426233640_create_users.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
+
+      t.timestamps
+    end
+  end
+end
```
Comment (Whoistolu) [inline db/migrate/20220426233640_create_users.rb]: - [ ] [REQUIRED] Kindly note that the `User` table according to the ERD diagram has the `posts_counter` . Kindly implement this.

File: db/migrate/20220427004448_create_comments.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+
+      t.timestamps
+    end
+  end
+end
```
Comment (Whoistolu) [inline db/migrate/20220427004448_create_comments.rb]: - [ ] [REQUIRED] Kindly add the necessary foreign keys to the `Comment` table which will create `author_id`, `post_id`, and a corresponding index. The foreign key is on the ERD diagram.

File: db/migrate/20220427004727_create_likes.rb [hunk 4]
```diff
@@ -0,0 +1,8 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+
+      t.timestamps
+    end
+  end
+end
```
Comment (Whoistolu) [inline db/migrate/20220427004727_create_likes.rb]: - [ ] [REQUIRED] Kindly add the necessary foreign key to the `Likes` table which will create `author_id`, `post_id`, and a corresponding index. The foreign key is on the ERD diagram.

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @oluyaratosin123 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AshakaE — APPROVED** [review]
Hi @oluyaratosin123  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**